src/commands/functions/functions-create.js

```diff
@@ -189,6 +203,51 @@ const pickTemplate = async function ({ language: languageFromFlag }) {
 
 const DEFAULT_PRIORITY = 999
 
+const selectTypeOfFunc = async () => {
+  const functionTypes = [
+    { name: 'Edge Function (Deno)', value: 'edge' },
+    { name: 'Serverless Function (Node/Go)', value: 'serverless' },
```

Comment:
[dust] I think we only use the capitalised name when referring to the feature (i.e. Edge Functions), not a function.

```suggestion
    { name: 'Serverless function (Node/Go)', value: 'serverless' },
```